app/src/main/java/com/example/pygmyhippo/common/Event.java

```diff
@@ -162,5 +177,23 @@ public EventStatus getEventStatus() {
     public void setEventStatus(EventStatus eventStatus) {
         this.eventStatus = eventStatus;
     }
+
+    // constructor to make event
+    public Event() {this.entrants = new ArrayList<>();}
+    public Event(String eventTitle, String eventID, String organiserID, ArrayList<Entrant> entrants, String location,
```

Comment:
I'm thinking that we could make eventID nullable since this would be something generated by the database.